play/apps/leaderboard/jobs/match_starter.py

```diff
@@ -31,10 +31,25 @@ def matches(self):
                 | Q(game__status=Game.Status.CREATED)
             )
         )
+
+        for snake_id in snake_ids:
+            s = Snake.objects.get(id=snake_id)
+            s.update_healthy()
+            leaderboardSnake = SnakeLeaderboard.objects.get(snake=s)
```

Comment:
python code would be `leaderboard_snake` this might be one of the things black is catching.

Reply:
This was not mentioned by black, but I fixed it anyway to be consistent in style.